src/main/java/com/bullhornsdk/data/model/entity/core/customobject/PersonCustomObjectInstance.java

```diff
@@ -0,0 +1,20 @@
+ package com.bullhornsdk.data.model.entity.core.customobject;
+
+ import com.bullhornsdk.data.model.entity.embedded.LinkedPerson;
+ import com.fasterxml.jackson.annotation.JsonProperty;
+
+ public abstract class PersonCustomObjectInstance extends CustomObjectInstance {
+
+     private LinkedPerson linkedPerson;
+
+     @JsonProperty("person")
+     public LinkedPerson getLinkedPerson() {
+         return linkedPerson;
+     }
+
+     @JsonProperty("person")
+     public void setLinkedPersoninkedPerson(LinkedPerson linkedPerson) {
```

Comment:
typo?  Also can we just name the property 'person' to be consistent with the JSON property?